src/main/java/WayofTime/alchemicalWizardry/common/rituals/RitualEffectWellOfSuffering.java

```diff
@@ -151,4 +168,11 @@ public List<RitualComponent> getRitualComponentList() {
         wellOfSufferingRitual.add(new RitualComponent(0, 1, -4, RitualComponent.AIR));
         return wellOfSufferingRitual;
     }
+
+    public int entityTypeMultiplier(EntityLivingBase entity) {
+        if (entity instanceof EntityVillager) return AlchemicalWizardry.VillagerMultiplierSacrificeWellOfSuffering;
```

Comment:
I still don't understand why you've just randomly threw villagers up to *12 for 0 reasoning